synthtool/gcp/templates/java_library/.github/dependabot.yml

```diff
@@ -11,4 +11,7 @@ updates:
     schedule:
       interval: "daily"
     # Disable version updates for pip dependencies
-    open-pull-requests-limit: 0
```

Comment:
This already says "# Disable version updates for pip dependencies"

Reply:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit

> This option has no impact on security updates, which have a separate, internal limit of ten open pull requests.

Reply:
I think the security category is how this is leaking through.

* https://github.com/googleapis/java-storage-nio/blob/main/.github/dependabot.yml#L14

Releases notes for the dependency list security in what was fixed.
* https://github.com/googleapis/java-storage-nio/pull/1180

Cryptography probably implicitly gets categorized as security
* https://github.com/googleapis/java-storage-nio/pull/1188

Reply:
Yes.